web/themes/custom/cloudy/src/components/_item-list.scss

```diff
@@ -61,7 +61,7 @@
 /* TOC ul styles */
 .toc-title {
     padding: .5rem 1rem;
-    background-image: linear-gradient(-60deg,$gold,$gold 25%,$primary 0);
+    background-image: linear-gradient(-60deg,$cloudy-gold-base,$cloudy-gold-base 25%,$primary 0);
```

Comment:
$primary is a color token from bootstrap, please replace with one of the theme variables starting with $cloudy-